service/storage_service.cc

```diff
@@ -3551,6 +3551,7 @@ db::schema_features storage_service::cluster_schema_features() const {
     f.set_if<db::schema_feature::VIEW_VIRTUAL_COLUMNS>(bool(_view_virtual_columns));
     f.set_if<db::schema_feature::DIGEST_INSENSITIVE_TO_EXPIRY>(bool(_digest_insensitive_to_expiry));
     f.set_if<db::schema_feature::COMPUTED_COLUMNS>(bool(_computed_columns));
+    f.set_if<db::schema_feature::CDC_OPTIONS>(bool(_cdc_feature));
```

Comment:
Zero commit log? What does this CDC_OPTIONS do? Why do we need it? It is not obvious to everyone.

Reply:
@haaawk

Reply:
not a part of this patchset anymore